docs/prover/cli/options.md

```diff
@@ -671,8 +671,21 @@ Prover.  `--settings` should be followed by a comma-separated list of options.
 This list is incomplete.
 ```
 
+(-optimisticFallback)=
+#### `--settings -optimisticFallback=true`
```

Comment:
Does this setting change the behavior of `AUTO` summaries (and is that all it does?)  If so, we should add a reference to the section on `AUTO` summaries, and also update that section to refer to this option.  You can add
```
{ref}`auto-summary` or {ref}`displayed text <auto-summary>`
```

to add a link